incubator/concourse/templates/worker-statefulset.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: {{ template "worker.fullname" . }}
+  labels:
+    app: {{ template "worker.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  serviceName: {{ template "worker.fullname" . }}
+  replicas: {{ .Values.worker.replicas }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "worker.fullname" . }}
+      annotations:
+        pod.alpha.kubernetes.io/initialized: "true"
```

Comment:
This annotation is no longer necessary for StatefulSet. It was only needed for the old incarnation called PetSet.